content/reference/current/installation/manual/index.md

```diff
@@ -11,11 +11,12 @@ weight: 1020
 
 * Your current Gatling Enterprise version must be at least 1.6.2. Otherwise, you must first upgrade to Gatling Enterprise 1.11.1.
 * Shut Gatling Enterprise process down
-* Perform a backup of your Cassandra data (eg `/var/lib/cassandra` directory, but might vary depending on how you've installed Cassandra)
+* Shut Cassandra down to perform a backup of your Cassandra data
+  _(eg `/var/lib/cassandra` directory, but might vary depending on how you've installed Cassandra)_
```

Comment:
Explain how to shutdown Cassandra.
If installed as a service, stop the service.
Otherwise, just kill the process.